src/Slices/DesiredState/UI/Components/Actions.tsx

```diff
@@ -20,6 +21,7 @@ export const Actions: React.FC<Props> = ({ version, isPromoteDisabled }) => {
       position="right"
       onSelect={() => setIsOpen(false)}
       dropdownItems={[
+        <DeleteAction key="delete" version={version} />,
```

Comment:
why is it cast to a Number in line 30, and not in the other places?

Reply:
unfortunately, I don't know, as these components were there from the start, maybe that made simpler to compare version to each other, but it's only a guess.
I'll try to dig and check what was the reasoning behind that :D